web/src/lib/components/asset-viewer/delete-button.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { type AssetResponseDto } from '@immich/sdk';
+
+import { assetFactory } from '@test-data/factories/asset-factory';
+import '@testing-library/jest-dom';
+import { render } from '@testing-library/svelte';
+import DeleteButton from './delete-button.svelte';
+
+export let asset: AssetResponseDto;
```

Comment:
No need to export this

Reply:
Are you sure? I want to declare a component prop. I get `TypeError: ctx[1] is undefined` if I remove the `export`.

Reply:
This is in the test file, not the component itself